vizro-core/src/vizro/static/css/layout.css

```diff
@@ -293,3 +293,9 @@ Current logic:
   flex-direction: row;
   gap: 12px;
 }
+
+.btn-circular {
+  border-radius: 50%;
+  height: 32px;
+  width: 32px;
```

Comment:
```suggestion
  height: 28px;
  width: 28px;
```
<img width="542" height="157" alt="Screenshot 2025-08-14 at 16 01 00" src="https://github.com/user-attachments/assets/eb034f04-604a-4d33-9d92-40dc290d3387" />


I think they looked a bit too big. Could you also make sure the collapse-icon has height/width of 28px then. Because optimally these two should looks the same in terms of dimension :)